modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/dataset_display/utilities/notifications.js

```diff
@@ -12,34 +12,24 @@
  * details.
  */
 
-export function showNotification(
-	message,
-	type = 'success',
-	closeable = true,
-	duration = 500
-) {
-	if (!window.AUI) {
-		return;
-	}
+import {openToast} from 'frontend-js-web';
+
+const TYPE_TITLE = {
+	danger: Liferay.Language.get('error'),
+	info: Liferay.Language.get('info'),
+	success: Liferay.Language.get('success'),
```

Comment:
This is missing `warning`, actually :)

Reply:
https://clayui.com/docs/components/alert.html


Property | Type | Required | Default | Description
-- | -- | -- | -- | --
displayType | 'danger' \| 'info' \| 'success' \| 'warning' | false | 'info' | Determines the style of the alert.

Reply:
Nevermind :)